### PR TITLE
增加array类型可以使用普通的逗号

### PR DESCRIPTION
**Pull request by 233cy**
增加array类型可以使用普通的逗号

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### CODE REVIEW

**Comment by 233cy** [comment]
为什么不通过呢。现在没有办法使用普通的逗号啊

File: lib/settings-panel.js [hunk 1]
```diff
@@ -323,7 +323,10 @@ export default class SettingsPanel extends CollapsibleSectionPanel {
         return floatValue
       }
     } else if (type === 'array') {
-      let arrayValue = (value || '').split(',')
+      let arrayValue = (value || '').split(/[^\\],/)
```
Comment (UziTech) [inline lib/settings-panel.js]: `.split(/[^\\],/)` will remove any character before a comma

```js
"1, 2".split(/[^\\],/) // ["", "2"]
```

I would still split on a comma and check if the last character in any group is a `\` then concatenate the groups

```suggestion
      let arrayValue = (value || '').split(',')
      arrayValue = arrayValue.reduce((values, val) => {
        const last = values.length - 1
        if (last >= 0 && values[last].endsWith("\\")) {
          values[last] = values[last].replace(/\\$/, ",") + val
        } else {
          values.push(val)
        }
        return values
      }, [])
```

**Comment by UziTech** [comment]
Do you want me to submit a new pull request?

你想让我提交新的拉取请求吗？

**Comment by 233cy** [comment]
> Do you want me to submit a new pull request?
> 
> 你想让我提交新的拉取请求吗？

是的，但是也没有通过。请问atom哪里可以看到内置插件呢？

**Comment by 233cy** [comment]
@defunkt 请问为什么通不过呢？只是增加使用普通逗号的功能

**Comment by UziTech** [comment]
I created a new PR #1102 that has passing tests.

**Comment by rsese** [comment]
Thanks @UziTech, going to close in favor of #1103.

And thanks for the pull request @233cy - again though, there's no need to @ mention specific people, we do our best to reply to issues and pull requests.  And as mentioned below, please use the templates for new issues and pull requests.

---

We require the template to be filled out on all new issues and pull requests. We do this so that we can be certain we have all the information we need to address your submission efficiently. This allows the maintainers to spend more time fixing bugs, implementing enhancements, and reviewing and merging pull requests.

Thanks for understanding and meeting us half way :grinning:

**Pull request closed without merging**